.travis.yml

```diff
@@ -5,6 +5,7 @@ sudo: false
 cache:
   directories:
     - $HOME/.composer/cache
+    - $HOME/.composer/vendor
```

Comment:
I'm not confident about this one. Keeping vendors forever can lead to hard to debug issues. Especially when vendors are corrupted or when Composer failed to update them properly and it happens when changing dependencies in `composer.json`. Sometimes, dependencies are not exactly the same when running a `composer update` with an existing `vendor/` directory and without it (the last example is when Zend Framework renamed the Zend Code directory).

Also, the build time will not be significantly improved as dependencies are already downloaded in the cache directory.
I've taken a quick look at Symfony, Laravel, and Zend Framework: they don't cache the `vendor/` directory.